content/02.body.md

```diff
@@ -139,8 +139,8 @@ Further discussion of this approximate edge prior and an derivation are availabl
 
 We performed three prediction tasks to assess the performance of the edge prior.
 We compared the permutation-based prior with two additional features: our analytical approximation of the edge prior and the product of source and target degree, scaled to the range [0, 1] to allow calibration assessment.
-We used 20 biomedical networks from the Hetionet heterogeneous network [@doi:10.7554/eLife.26726] that had at least 2000 edges for the first two tasks.
-In the first task, we computed the degree-based prediction features (edge prior, scaled degree product, and analytical prior approximation), and predicted the original edges in the network.
+We used 20 biomedical networks from the Hetionet heterogeneous network [@doi:10.7554/eLife.26726] that had at least 2000 edges for the first two tasks ([Supplemental table](#networks)).
+In the first task, we computed the degree-based prediction features (edge prior, scaled degree product, and analytical prior approximation), and predicted the original edges in the network by rank-ordering node pair edge predictions by the node pairs' feature values.
```

Comment:
I'm not sure I understand what "by rank-ordering node pair edge predictions by the node pairs' feature values." The phrasing is really hard to parse, and you say that you "predicted...by ordering predictions," which feels recursive to me.